### PR TITLE
Adds the parser for ConfigMap

### PR DESCRIPTION
**Pull request by Nithish555**
Adds a attribute for CloudSyncInterval
Deletes the ConfigMap webhook and controller
Adds a parser for the ConfigMap attributes

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -29,6 +30,8 @@ require (
 	sigs.k8s.io/controller-runtime v0.12.2
 )
 
+require github.com/pmezard/go-difflib v1.0.0 // indirect
```
Comment (Anandkumar26) [inline go.mod:33]: Add this require block in the indirect dependency (L36) and run make.

File: cmd/nephe-controller/options_test.go [hunk 2]
```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"antrea.io/nephe/pkg/config"
```
Comment (Anandkumar26) [inline cmd/nephe-controller/options_test.go:4]: fix imports

File: go.mod (hunk 1 above)
Comment (Nithish555) [reply go.mod]: Solved.

File: cmd/nephe-controller/options_test.go (hunk 2 above)
Comment (Nithish555) [reply cmd/nephe-controller/options_test.go]: Solved.

File: pkg/controllers/cloud/networkpolicy_controller.go [hunk 3]
```diff
@@ -589,8 +592,9 @@ func (r *NetworkPolicyReconciler) Start(stop context.Context) error {
 		case <-ticker.C:
 			r.backgroupProcess()
 			r.retryQueue.CheckToRun()
-			if time.Now().Unix()&cloudSyncInterval == 0 {
+			if time.Now().Unix()-lastSyncTime >= r.CloudSyncInterval {
```
[the remark this answers is not in the document]
Comment (Nithish555) [reply pkg/controllers/cloud/networkpolicy_controller.go:595]: Golang IDE prefers this. If we there is extra space it automatically delete.

File: cmd/nephe-controller/options.go [hunk 4]
```diff
@@ -0,0 +1,75 @@
+package main
```
[the remark this answers is not in the document]
Comment (Nithish555) [reply cmd/nephe-controller/options.go:15]: Solved.

File: config/manager/nephe-controller.yaml [hunk 5]
```diff
@@ -25,7 +25,9 @@ metadata:
 data:
   nephe-controller.conf: |
     # Specifies the prefix to be used while creating cloud resources.
-    # CloudResourcePrefix: nephe
+    #CloudResourcePrefix: nephe
```
[the remark this answers is not in the document]
Comment (Nithish555) [reply config/manager/nephe-controller.yaml]: Solved.

File: pkg/config/config.go [hunk 6]
```diff
@@ -0,0 +1,12 @@
+package config
```
[the remark this answers is not in the document]
Comment (Nithish555) [reply pkg/config/config.go:15]: Solved.

**Review by reachjainrahul — APPROVED** [review]
/LGTM

**Pull request merged**